pydrive2/fs/spec.py

```diff
@@ -542,6 +542,35 @@ def cp_file(self, lpath, rpath, **kwargs):
             buffer = io.BytesIO(stream.read())
             self.upload_fobj(buffer, rpath)
 
+    @_gdrive_retry
+    def mv(self, path1, path2, recursive=False, maxdepth=None, **kwargs):
+
+        if recursive:
+            raise NotImplementedError("Recursive move is not supported")
+        if maxdepth:
+            raise NotImplementedError("Max depth move is not supported")
+
+        file1_name = posixpath.basename(path1)
+        file2_name = posixpath.basename(path2)
+
+        file1_parent = self._parent(path1)
+        file2_parent = self._parent(path2)
+
+        file1_id = self._get_item_id(path1)
+
+        file1 = self.client.CreateFile({"id": file1_id})
+
+        file1.FetchMetadata(fields="title,parents")
```

Comment:
https://github.com/iterative/PyDrive2/pull/223 should fix this, please rebase and let's try to simplify this